app/routes/meetings/MeetingDetailRoute.js

```diff
@@ -31,15 +35,21 @@ const loadData = (props, dispatch) => {
 };
 
 const mapStateToProps = (state, props) => {
+  const { params: { meetingId }, currentUser } = props;
   const { action, token } = props.location.query;
   const meetingsToken = state.meetingsToken;
+  const meeting = selectMeetingById(state, { meetingId });
+  const comments = selectCommentsForMeeting(state, { meetingId });
```

Comment:
You should move this "logic" to `MeetingDetailLoginRoute`. This route is used when a user isn't authenticated, and opens an "accept invitation" link.

Reply:
The whole const mapStateToProps block, or just the new additions?

Reply:
Only `comments`. The meeting is selected in `MeetingDetailLoginRoute` already